### PR TITLE
Document the tree restructuring

### PR DESCRIPTION
**Pull request by mvidner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,30 @@ things get gemified, packaged, etc.
...
+tictactoe-server
+├── doc               ->  /usr/share/doc/packages/yast2-ticactoe-server
+├── src
+│   ├── clients       ->  /usr/share/YaST2/clients
+│   ├── include       ->  /usr/share/YaST2/include
+│   ├── modules       ->  /usr/share/YaST2/modules
+│   ├── scrconf       ->  /usr/share/YaST2/scrconf
+│   ├── autoyast-rnc  ->  /usr/share/YaST2/schema/autoyast/rnc
+│   ├── control-rnc   ->  /usr/share/YaST2/schema/control/rnc
+│   └── desktop       ->  /usr/share/applications/YaST2
```
Comment (jreidinger) [inline README.md]: There is missing data dir ( e.g. sysconfig use it ) and  bin ( e.g. sysconfig use it, goes to /usr/lib/YaST2/bin/ )

File: README.md [hunk 2]
```diff
@@ -83,6 +83,30 @@ things get gemified, packaged, etc.
...
+The existing directory tree layout of nearly all YaST modules is rather random
+and stupid. It is convenient to use the translation occasion to also move
+files to a more logical and uniform scheme, to enable exporting the working
+directory via `Y2DIR` and to unify Makefiles.
+
+The following tree shows what gets installed where:
+
+```
+tictactoe-server
+├── doc               ->  /usr/share/doc/packages/yast2-ticactoe-server
```
Comment (jreidinger) [inline README.md]: actually we do not care about others like doc or testsuite now. We just reformat stuff in src or exported dirs ( like library in yast2 )

**Comment by dmajda** [comment]
LGTM

@jreidinger If you don't have any comment to this, please merge.

File: README.md [hunk 3]
```diff
@@ -167,3 +234,65 @@ Total ERROR(ruby):  0
...
+# Default: []
+moves:
+    # a glob in the original structure
+    # (use quotes because of YAML syntax).
+  - from: src/NfsServer.ycp
+    # a directory path in the new structure.
+    # Missing directories will be created.
+    to: src/modules
+  - from: "src/nfs[-_]server*.ycp"
+    to: src/clients
```
Comment (jreidinger) [inline README.md:276]: this is nice place to document that moves is order dependent

**Pull request merged**